src/sources/nz.json

```diff
@@ -0,0 +1,15 @@
+[
+    {
+        "url": "http://data.ecan.govt.nz/data/29/Air/Air%20quality%20data%20for%20a%20monitored%20site%20(10%20Minute)/JSON?SiteID=$station&StartDate=$date&EndDate=$date",
+        "adapter": "canterbury",
+        "name": "New Zealand Canterbury",
```

Comment:
use a more descriptive name for this. What is the name of the data provider instead of where it covers